### ISSUE TITLE
There is no program menu in the right click menu in file explore in Windows11

### ISSUE THREAD

**Issue by wwwwwalter** [Tichau/FileConverter#403]
**Describe the bug**
There is no program menu in the right click menu in file explore in Windows11.
not open the program when choose it in file system

** window env**
22H2(OS 22621.3155)

**Comment by Tsaitou7361** [comment]
Push this!

**Comment by Tichau** [comment]
Hello,
Can you give me more info about your issue:
- Is installation succeeded? Can you find the files where you installed it?
- If you try to launch it directly, it should display an help gif, is it working ?

**Comment by wwwwwalter** [comment]
Yes！I can see the git in the main program，but it doesn't lunched when i choose it from "Choose other application" through right click menu.I will record a video for you. Wait a moment. Thank you for your help！

---- Replied Message ----
| From | Adrien ***@***.***> |
| Date | 03/13/2024 16:59 |
| To | Tichau/FileConverter ***@***.***> |
| Cc | ***@***.***>,
Author ***@***.***> |
| Subject | Re: [Tichau/FileConverter] There is no program menu in the right click menu in file explore in Windows11 (Issue #403) |

Hello,
Can you give me more info about your issue:

Is installation succeeded? Can you find the files where you installed it?
If you try to launch it directly, it should display an help gif, is it working ?

**Comment by wwwwwalter** [comment]
Hi,I have recored two videos. I have put them in attachment.
vedio-1.new reinstall
vedio-2.use it

https://github.com/Tichau/FileConverter/assets/41465754/9684090a-7242-4bf7-b9a6-2d4caa4a5714


https://github.com/Tichau/FileConverter/assets/41465754/3ae4df12-4d29-4c70-aadb-970690ec342f



Thanks again

**Comment by wwwwwalter** [comment]
Oh! It's OK,I see the menu in system menu,when I restart Explore.exe.
I think there are no bugs(1.on menu 2.not lunch) in FileConverter.Because I test it with ".ncm" file  yesterday which is not supported.Will it support ".ncm" file download from "网易云音乐" in future?

Thank you for your help

https://github.com/Tichau/FileConverter/assets/41465754/934ac808-c2ec-4b51-8ba8-6c08addb8a6e

**Comment by AIV-DEV** [comment]
it's also in win10 22H2, build 19045.4046
also settings not launch.
help gif work.

**Comment by Tichau** [comment]
Yes, the menu does appear only on supported file types to not clutter the explorer menu.

@AIV-DEV can't you give me more information please ?

**Comment by AIV-DEV** [comment]
What information?
Does the software save log files somewhere?
(And thank you for all the work!)

**Comment by Tichau** [comment]
Information are described here: https://github.com/Tichau/FileConverter/wiki/Troubleshooting#report-an-issue

**Comment by AIV-DEV** [comment]
OK.
this all files from 
```
%localappdata%\FileConverter\
```

**Comment by Underbares** [comment]
Installed fileconverter.2.0.2 on W10 over fileconverter.1.2.3.
Explorer Menu does not show fileconverter entry anymore.
Removed 2.0.2 and reinstaller 1.2.3 and enry is back in the menu.
Without entry this great tool is useless. Can you plz fix this?
